src/bci_build/package/base.py

```diff
@@ -93,6 +94,18 @@ def _get_base_config_sh_script(os_version: OsVersion) -> str:
     ).render(os_version=os_version)
 
 
+@dataclass
+class Sles15Image(OsContainer):
+    @property
+    def build_tags(self) -> list[str]:
+        return list(
+            set(
+                super().build_tags
+                + ["suse/sle15:%OS_VERSION_ID_SP%", f"suse/sle15:{self.version_label}"]
```

Comment:
this changes the first tag to be `tag="%OS_VERSION_ID_SP%.%RELEASE%"` rather than `tag="%OS_VERSION_ID_SP%"`. 

I'm not sure how important that is to preserve, but it is a behavior change (I think it will lead to a different buildversion, which might have funky side effects.